src/SumoCoders/FrameworkCoreBundle/Resources/assets/sass/components/_datepicker.scss

```diff
@@ -1,3 +1,19 @@
+.datepicker {
+  z-index: 1000000 !important;
```

Comment:
yuk, are the frontenders oke with the !importants?

Reply:
fixed 22aa43c